.github/workflows/run.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           - 'basic || networks/typical::istanbulmps'
           - 'mps-upgrade-txtrace || networks/template::raft-4nodes-mps'
           - 'mps-upgrade-txtrace || networks/template::istanbul-4nodes-mps'
-          - '(basic && !nosupport && !mps && (spam && raw) && !eth-api-signed && !privacy-enhancements-disabled && !graphql && !async && !extension && !storage-root && !get-quorum-payload && !personal-api-signed) || networks/typical-besu::ibft2'
+          - '(basic && !nosupport && !mps && !(spam && !raw) && !eth-api-signed && !privacy-enhancements-disabled && !graphql && !async && !extension && !storage-root && !personal-api-signed) || networks/typical-besu::ibft2'
```

Comment:
!(spam && !raw) is hard to read
(!spam || raw) is equivalent, if I don't mistake

Reply:
that is different, as will run any `raw` test.

I want to not run the ones that are `spam` but not `raw`.

this is to fix an issue now, as currently is running only the spam with raw